### DISCUSSION TITLE
📣 Help improve GitHub dependency graph with your feedback!

### DISCUSSION THREAD

**Discussion by tassia-sp** [community/community#43364] [Code Security]
Hello everybody 👋 The GitHub Security Products team needs your help defining the future of the dependency graph. We're scheduling 60-minute Zoom interviews with developers January 9-13. You'll be compensated with a $125 gratuity. Before scheduling an interview, we have just a few questions. Please [fill out our quick survey here](https://github.surveymonkey.com/r/YBG57CV). We’ll reach out if you’re a fit for the study. Thank you! 🙏

**Comment by ckipp01** [comment]
Hey @tassia-sp I'm wondering if this is still a possibility? I see the dates have past, but as far as I know no one actively working in the Scala Tooling system (including the sbt and Mill plugins) was actively involved. If it's at all still a possibility, I'd (on behalf of the [Scala Center](https://scala.epfl.ch/)) love to provide some feedback. I did fill out the survey as well.

**Comment by marr-cloud** [comment]
one improvement I would like to see in the dependency graph is the possibility to see not only the dependencies of the main branch but also of the other branches and to link it with Dependabot to have a better control.
